extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -95,11 +95,16 @@ export class AmpSelector extends AMP.BaseElement {
     if (!this.isMultiple_) {
       selectedArray = selectedArray.slice(0, 1);
     }
+    // Convert array values to strings and create map for fast lookup.
+    const selectedMap = selectedArray.reduce((map, value) => {
+      map[value] = true;
+      return map;
+    }, Object.create(null));
```

Comment:
wouldn't they always be strings since they are read from the dom ? also where is the string conversion happening?